source/sass/global.scss

```diff
@@ -6,31 +6,6 @@ body {
     background: $white;
   }
 
-  // Subpages
```

Comment:
Renamed the vague selector name from `.wrapper-2` to `#hero` and cleaned up DOM a bit so SCSS portion got updated too.